packages/core/src/amazonq/lsp/lspController.ts

```diff
@@ -133,7 +133,7 @@ export class LspController {
     }
 
     async getFileSha384(filePath: string): Promise<string> {
-        const fileBuffer = await fs.readFile(filePath)
+        const fileBuffer = await fs.readFileBytes(filePath)
```

Comment:
if we are also renaming this one, then maybe the "text" version could simply be `readFile` ?